INN/umbrella-workdayminn
# Display all categories on post instead of only top term

## Changes

This pull request makes the following changes:

- Created `workday_get_post_categories_with_top_term` function that grabs all categories from a post + top term, moves the top term to the first item in the array, and formats all categories inside of a link to be displayed.
- Utilizes new function on single post page header now instead of the `largo_maybe_top_term` function to display all post categories assigned.

![Screen Shot 2019-08-13 at 2 18 22 PM](https://user-images.githubusercontent.com/18353636/62966815-d2925580-bdd5-11e9-9205-fa6e3897488a.png)

## Why



For #64

## Testing/Questions

Features that this PR affects:

- Single post categories displaying


Questions that need to be answered before merging:

- [ ] Is it too convoluted/is there an easier way to do this?

Steps to test this PR:

1. Add categories to a post
2. Assign a top term
3. Verify the top term is the first category displayed
4. Make sure it looks ok
